CHANGELOG.md

```diff
@@ -87,4 +87,18 @@ The format is (loosely) based on [Keep a Changelog](http://keepachangelog.com/)
 - Best practices check for too many links in object
 - Best practices check for summaries in collections
 - Validation from stac-validator 2.3.0  
-- Links and assets validation checks  
+- Links and assets validation checks    
+
+-------  
+### Compare Releases  
+  
```

Comment:
```suggestion
```

No header content needed, these links are not rendered.

Reply:
ah ok